.changeset/rotten-colts-decide.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-catalog-backend-module-incremental-ingestion': minor
+---
+
+Allow incremental event handlers to be async; Force event handler to indicate if
```

Comment:
Can we prefix this with `**BREAKING**` and show a code example for before and after for the `onEvent` method?